data/human/weapons.txt

```diff
@@ -1005,6 +1005,36 @@ outfit "Javelin Mini Pod"
 	description "	The weapon's high rate of fire also makes it useful for saturating well-protected ships' anti-missile defenses so that more dangerous projectiles fired by allied ships can sneak through."
 	description "	Mini pods enable fighters and other light craft to carry small quantities of Javelins into battle while retaining enough space for other weapons. Large ships are better off with full-size pods."
 
+outfit "Javelin Turret"
+	category "Secondary Weapons"
+	cost 110000
+	thumbnail "outfit/javelin turret"
+	"mass" 31
```

Comment:
```suggestion
	"mass" 15
```
As 400 javelins have a total of 16 mass, this way the turret's mass matches its outfit space when fully loaded. That's the standard for missile weapons.